app/views/course/statistics/aggregate/submission_due.json.jbuilder

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+json.name @all_students.first.name
+
+json.assessments @assessments do |assessment|
+  student_id = @all_students.first.id
+  submission_info = @submissions[[assessment.id, student_id]]
+  timeline = @personal_timeline[[assessment.id, student_id]] || @reference_timeline[assessment.id]
+  _, reference_end_at, = @reference_timeline[assessment.id]
+
+  start_at, end_at, is_fixed = timeline
+
+  if submission_info
+    submission_id, workflow_state, = submission_info
+
+    next if workflow_state != 'attempting'
+
+    json.submissionId submission_id
+    json.workflowState workflow_state
+  else
+    is_not_released_to_student = Time.now.to_i < start_at.to_i
+
+    next if is_not_released_to_student
+
+    json.workflowState 'unstarted'
+  end
+
+  json.dueIn time_until_due(Time.now, end_at)
```

Comment:
might be sufficient but I'm afraid the code's not communicative enough (right now it's just using `Time.now` since we are not considering the submitted/graded/published submissions).

But for the future, if we want to consider those kinds of submissions as well, the function `time_until_due` might be more general and can be used for that case as well.